src/spec/apis/funds_api.py

```diff
@@ -142,10 +144,24 @@ async def list_historical_values_spec(
     ) -> List[HistoricalValue]:
         """Lists historical values"""
         return await self.list_historical_values(
-            fundId,
+            self.toUuid(fundId),
             first_result,
             max_results,
             start_date,
             end_date,
             token_bearerAuth
         )
+
+    def toUuid(self, str: str) -> UUID:
+        """Translates str to UUID
+
+        Args:
+            str (str): str
+
+        Returns:
+            UUID: UUID
+        """
+        try:
+          return UUID(str)
+        except ValueError:
+          raise HTTPException(status_code=400, detail="Invalid UUID {str}".format(str = str))
```

Comment:
please insert one tab before raise, also remove extra space for str = str -> str=str